go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/inspektor-gadget/inspektor-gadget
 
-go 1.21
+go 1.21.0
```

Comment:
Why do we need this change?

Reply:
Slack discussion: https://cilium.slack.com/archives/C027KBX679U/p1708072798219249 & https://github.com/golang/go/issues/62278